modules/core/src/main/scala/org/tessellation/l0/domain/snapshot/SnapshotStorage.scala

```diff
@@ -1,7 +1,11 @@
 package org.tessellation.l0.domain.snapshot
+import org.tessellation.kernel.StateChannelSnapshot
 import org.tessellation.schema.height.Height
 
 trait SnapshotStorage[F[_]] {
   def storeSnapshot(snapshot: GlobalSnapshot): F[Unit]
   def getLastSnapshotHeight: F[Height]
+  def getLastSnapshotSubHeight: F[Height]
+  def getLastSnapshotOrdinal: F[GlobalSnapshotOrdinal]
+  def getStateChannelSnapshots: F[Set[StateChannelSnapshot]]
```

Comment:
shouldn't it be a `Map[StateChannelAddress, Set[StateChannelSnapshot]]` or `def getStateChannelSnapshots(address: StateChannelAddress): F[Set[StateChannelSnapshot]]` or sth like that?

Reply:
I am not sure in which form we should store state-channel snapshots. Maybe you are right.
In general I just need to store statechannel snapshots, then put them to Snapshot and clear the storage by removing these which are in Snapshot already (or maybe in Majority Snapshot)